docs/resources/azure_dns_zones_resources.md

```diff
@@ -0,0 +1,98 @@
+---
+title: About the azure_dns_zones_resources Resource
+platform: azure
+  ---
+
+# azure_dns_zones_resources
+
+Use the `azure_dns_zones_resources` InSpec audit resource to test properties related to bastion hots for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/dns/zones/get) for  properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+## Syntax
+
+An `azure_dns_zones_resources` resource block returns all Azure bastion hots, either within a Resource Group (if provided)
+  ```ruby
+  describe azure_dns_zones_resources(resource_group: 'my-rg') do
```

Comment:
Is `resource_group` required?

Reply:
Yes Ian .It is needed .Mentioning the api below .
GET https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{**resourceGroupName**}/providers/Microsoft.Network/dnsZones/{zoneName}?api-version=2018-05-01

Reply:
No, Ian .My mistake .updating the doc .